browser/banners/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+if (!is_android) {
```

Comment:
Maybe we don't need this check as this `browser_tests` target is already filtered in `test/BUILD.gn` ?